.github/workflows/main-arm64.yaml

```
@@ -45,6 +45,10 @@ jobs:
     name: Multiple build
     needs: image-prepare
     steps:
+        # this step should be deleted once CNCF hosted/ephemeral runners are in use
```

```suggestion
        # TODO: this step should be deleted once CNCF hosted/ephemeral runners are in use
```
Thsnks for the clarification, and it's better add a todo here. BTW when will the CNCF hosted/ephemeral runners be used?  :)